test_client/src/main/scala/io/reactivesocket/tck/ClientDSL.scala

```diff
@@ -74,16 +74,13 @@ class ClientDSL {
 
 object clienttest extends ClientDSL {
   def main(args: Array[String]) {
-    Tests.runTests(this)
-    end()
+    Tests.runTests(this, this.writer)
   }
 
   // example for testing channel
   @Test
-  def test0() : Unit = {
-    begintest()
-    nametest("channel test")
-    requestChannel using("a", "b") asFollows(() => {
+  def channel_test() : Unit = {
```

Comment:
I'll comment just here, but seen it around a few times.
Scala style is definitely not snake case, it's normal java method naming style - let's stick to that unless good reasons to deviate from it.